reference/docs-conceptual/community/contributing/editorial-checklist.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Editorial checklist
+description: This is a summarized list of rules for editing PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Editor's checklist
+
+This is a summary of rules to apply when writing new or updating existing articles. See other
+articles in the Contributor's Guide for detailed explanations and examples of these rules.
+
+## Metadata
+
+- `ms.date`: must be in the format **MM/DD/YYYY**
+  - Change the date when there is a significant or factual update
+    - Reorganizing the article
+    - Fixing factual errors
+    - Adding new information
+  - Do not change the date if the update is insignificant
+    - Fixing typos and formatting
+- `title`: unique string of 43-59 chars including spaces
+  - Do not include site identifier (it is auto-generated)
+  - Use sentence case - capitalize only the first word and any proper nouns
+- `description`: 115-145 characters including spaces - this abstract displays in the search result
```

Comment:
Based on this, most of these new articles should have longer descriptions. 😄

Reply:
stet